app.js

```diff
@@ -49,12 +49,8 @@ app.get('/', function (req, res) {
       res.write('<h1>Welcome to the Globalization Pipeline Node.js sample!</h1>');
       res.write('<h2> Pick a language code:</h2>');
       res.write('<ul>');
-      // add the source (probably english)
-      addLang(projInfo.sourceLanguage);
-      // add all of the targets
-      if( projInfo.targetLanguages) {
-        projInfo.targetLanguages.forEach( addLang );
-      }
+      projInfo.languages() // --> Array: [ 'en', 'de', … ]
```

Comment:
* `projInfo` is now a `Bundle` object here
* the new `languages()` function returns both the source AND target languages, in a single convenient array!